foods/services/FoodServices.py

```diff
@@ -0,0 +1,37 @@
+from django.http import Http404
+from ..models.FoodModel import Food
+from ..models.ImagesMopdel import Images
+from ..models.CategoryModel import Category
+
+class FoodService:
+
+    @classmethod
+    def save_img(cls, images, food):
+        for file in images:
+            Images.objects.create(image_url=file, food_id=food)
+
+    @classmethod
+    def save_food(cls, data):
+        food = Food.objects.create(
+            name=data.get('name'),
+            category_id=data.get('category_id'),
+            price=data.get('price'),
+            discount=data.get('discount'),
+            description=data.get('description'),
+        )
+        cls.save_img(data.files, food)
+        return food
+    @classmethod
+    def update_food(cls, id, data):
+        try:
+            food = Food.objects.get(id=id)
+            food.name = data.get('name')
+            food.category_id = data.get('category_id')
+            food.price = data.get('price')
+            food.discount = data.get('discount')
+            food.description = data.get('description')
+            cls.save_img(data.files, food)
```

Comment:
what's happened when save images is a success but food errors?

Reply:
Yes, I will adjust add the food first